contrib/vagrant-docker/README.md

```diff
@@ -31,7 +31,7 @@ stop on runlevel [!2345]
 respawn
 
 script
-    /usr/bin/docker -d -H=tcp://0.0.0.0:2375
+    /usr/bin/docker -H=tcp://0.0.0.0:2375 daemon
```

Comment:
Aside from the technical aspect, IMHO the `=arg`syntax is confusing when used on a short option. It is very common for the long form options but I cannot recall having seen this elsewhere before.
I recommend using either `-H tcp://0.0.0.0:2375` or `--host=tcp://0.0.0.0:2375`